### PR TITLE
Added suggested changes to SQLi section

### PR DESCRIPTION
**Pull request by Aroson1**

### CODE REVIEW

**Comment by errorxyz** [comment]
For ref. continuation of https://github.com/CSYClubIIITK/CTF-Guide/pull/4

File: src/web-exploitation/intro-to-sqli/README.md [hunk 1]
```diff
@@ -17,40 +17,81 @@ SQL Injection (SQLi) is a security vulnerability that allows attackers to inject
 7. [Second Order Injections](second-order-injections.md)
 
 ## Basis to Tackle SQLi Challenges
```
Comment (errorxyz) [inline src/web-exploitation/intro-to-sqli/README.md:19]:
```suggestion
## Basics of SQL
```

File: src/web-exploitation/intro-to-sqli/README.md [hunk 2]
```diff
@@ -17,40 +17,81 @@ SQL Injection (SQLi) is a security vulnerability that allows attackers to inject
...
-* The `--` comments out the rest of the query, effectively bypassing the password check and logging the attacker in as any user.
-* Basic SQL syntax
-  1. `SELECT` Extracts data from the given table.
-  2. `ORDER BY` Orders the results gotten from SELECT in a specific manner. For example, if one has a table of countries and their populations. One can select the countries starting with the letter R and then order them by their population.
-  3. `UNION` clause allows you to group multiple queries together.
-  4. `JOIN` Joins data from two tables depending on a certain characteristic on the table. So if a theres two table one with customer IDs and their addresses and another table with customer IDs and their purchases. You can join both tables so that the customer addresses match their purchases.
-  5. `DELETE, INSERT` Allows you to delete data or add new data to a table.
-  6. `AND,OR` Allows you to modify queries so that they return information depending on multiple categories.
-  7. `MIN,MAX` They return the smallest or largest value of a query
 * Look for places from where data might be used to construct a query. Like a textbox, or the url of the website.
```
Comment (errorxyz) [inline src/web-exploitation/intro-to-sqli/README.md:20]:
```suggestion
* User input from textboxes, URL parameters, etc may be used to construct a SQL query in the backend
```
add this point at the end and it'd be nice to have a small example to improve clarity and explain what happens in the backend at a high level

File: src/web-exploitation/intro-to-sqli/blind-injections.md [hunk 3]
```diff
@@ -56,32 +78,33 @@ Hint: First use auth by-pass then start applying blind SQL.
 This is based on a real CTF question, from the RaziCTF 2020.
 In this question you are given a login page and a `/login` endpoint we can post credentials to. When trying to log in it just says `You are not allowed to login` which seems quite odd. 
 ```
-http://130.185.122.155:8080/login 
+http://example.com/login
```
Comment (errorxyz) [inline src/web-exploitation/intro-to-sqli/blind-injections.md:81]: Doesn't make sense to explain a challenge if the user can't access it.
From the user's POV, this section is supposed to be practical and not another reading exercise.
This should apply to the other articles as well

**Review by errorxyz — CHANGES_REQUESTED** [review]

**Review by Kanishk-mittal — APPROVED** [review]

**Comment by errorxyz** [comment]
continued in #15

**Pull request closed without merging**